packages/dnb-eufemia/src/components/tooltip/TooltipContainer.tsx

```diff
@@ -221,12 +221,8 @@ export default function TooltipContainer(
       },
     }
 
-    if (stylesFromPosition[position]) {
-      stylesFromPosition[position]()
-    }
-    if (stylesFromArrow[arrow]) {
-      stylesFromArrow[arrow]()
-    }
+    stylesFromPosition?.[position]?.()
+    stylesFromArrow?.[arrow]?.()
```

Comment:
```suggestion
    stylesFromArrow[arrow]?.()
```

`stylesFromArrow` is always defined and an object.